microsoft-365/enterprise/configure-exchange-server-for-hybrid-modern-authentication.md

```diff
@@ -146,6 +149,16 @@ Set-AuthServer -Identity EvoSTS -IsDefaultAuthorizationEndpoint $true
 Set-OrganizationConfig -OAuth2ClientProfileEnabled $true
 ```
 
+If the EXCH version is Exchange 2016 [CU18 or higher] or Exchange 2019 [CU7 or higher] & Hybrid was configured with HCW downloaded after September 2020, run the following command in the Exchange Management Shell, on-premises:
+
+```powershell
+Set-AuthServer -Identity "EvoSTS - {GUID}" -Domain "Tenant Domain" -IsDefaultAuthorizationEndpoint $true
+Set-OrganizationConfig -OAuth2ClientProfileEnabled $true
+```
+
+> [!NOTE]
+> In case, EXCH is in Hybrid with **multiple Tenants**, there are multiple AuthServer Objects present in EXCH with Domains corresponding to each Tenant.  **IsDefaultAuthorizationEndpoint** Flag should be set true (using cmdlet 1 above) for any one these Authserver object. This flag can't be set true for all the Authserver objects and HMA would be enabled even if one of these AuthServer object's **IsDefaultAuthorizationEndpoint** flag is set to true.
```

Comment:
```suggestion
> In case EXCH is in hybrid with **multiple tenants**, there are multiple AuthServer objects present in EXCH with domains corresponding to each tenant.  The **IsDefaultAuthorizationEndpoint** flag should be set to true (using cmdlet 1 above) for any one of these Authserver objects. This flag can't be set to true for all the Authserver objects and HMA would be enabled even if one of these AuthServer object's **IsDefaultAuthorizationEndpoint** flag is set to true.
```